README.md

```diff
@@ -71,6 +71,14 @@ We recommend you read the "Architecture" section in our
 [White Paper](https://github.com/artipie/white-paper) to fully
 understand how Artipie is designed.
 
+## Contents
+
+- [Multitenancy](#multitenancy)
+- [Repository permissions](#repository-permissions)
+- [Storage configuration](#storage-configuration)
+- [Metrics](#metrics)
+- [Artipie REST API](#artipie-rest-api)
```

Comment:
@olenagerasimova I think we need some adjustements to order of items according to importance of feature to users:

- "Storage configuration" should come first - it's most important part, every user will need to configure storage I believe.
- Then I'd put "Repository permissions" before "Multitenancy", because configuring permissions is usually required when there more then one team is working with Artipie or it is available to public users from outside of organization. "Multitenancy" is feature for even larger installations when there are even there are independent parties are using the same Artipie installation, such as `central.artipie.com`.

So the following order in my opinion will make more sense:

- Storage configuration
- Repository permissions
- Multitenancy
- Metrics
- Artipie REST API